src/pymodaq_gui/utils/widgets/tree_toml.py

```diff
@@ -72,7 +72,10 @@ def param_to_dict(cls, param: Parameter) -> dict:
                     pdt = datetime.fromtimestamp(qdt.toSecsSinceEpoch())
```

Comment:
I just saw that there is a wrong import here it should be datetime.datetime.fromtimestamp (same also for line 67)